sponsor/urls.py

```diff
@@ -1,12 +1,12 @@
-from django.urls import path
-from django.contrib.auth.decorators import login_required
-
-from sponsor.views import SponsorRequestView
-
-app_name = 'sponsor'
-urlpatterns = [
-    path(
-        'sponsor-request/', login_required(SponsorRequestView.as_view()),
-        name='sponsor-request'
-    ),
-]
+# from django.urls import path
+# from django.contrib.auth.decorators import login_required
+#
+# from sponsor.views import SponsorRequestView
+#
+# app_name = 'sponsor'
+# urlpatterns = [
+#     path(
+#         'sponsor-request/', login_required(SponsorRequestView.as_view()),
+#         name='sponsor-request'
+#     ),
+# ]
```

Comment:
@amakarudze Yeah the URLs aren't loaded by the project, so by including this, it causes the coverage score to drop even though this URL can never be accessed because the site isn't including it.

You can see this here; https://github.com/DjangoGirls/djangogirls/blob/main/djangogirls/urls.py#L45